src/pages/VenuePage/VenuePage.tsx

```diff
@@ -79,16 +79,17 @@ const VenuePage: React.FC = () => {
     isUserPurchaseHistoryRequestedSelector
   );
 
-  // @debt we REALLY shouldn't be loading all of the venues collection data like this, can we remove it?
-  useFirestoreConnect("venues");
-
   const userId = user?.uid;
 
   const venueName = venue?.name ?? "";
   const venueTemplate = venue?.template;
 
   const event = currentEvent?.[0];
 
+  const { isRelatedVenuesLoading } = useRelatedVenues({
+    currentVenueId: venueId,
+  });
+
```

Comment:
It seems the only thing `useRelatedVenues` is used for in this component is `isRelatedVenuesLoading`. I'm not sure that that's something we need to necessarily have here..  and if we can remove it I expect that will mean that:

- we can move `RelatedVenuesProvider` back into `TemplateWrapper` where it used to be + where I prefer it
- it will most likely increase the perceived loading performance of the app, as we no longer will need to block the rest of the app based on this being available (and if there is something 'deeper down' that needs it, then best for that component to handle its own 'loading' rather than up here)

Reply:
Hopefully this is possible:
```suggestion
```

Reply:
Done in 4908ac4abd39131d79fdaedea96781e81ebae0cb